data/membership-schema/src/main/resources/net/corda/schema/membership/member/static/registration/1.0/corda.member.static.registration.json

```diff
@@ -17,6 +17,11 @@
       "description": "Version supported for the flow protocol used by the notary service. Valid only when one of the roles is notary.",
       "type": "integer",
       "minimum": 1
+    },
+    "^ext\\.[a-zA-Z0-9.]+$": {
+      "description": "Optional. A user can specify additional properties, they must be prefixed with `ext.`.",
+      "type": "string",
+      "maxLength": 256
```

Comment:
I think one requiremenent we have for this is users of this feature are going to be using this for building permissioning schemes based off cryptographic keys, right? Are they going to include PEM-formatted public keys in this attributes and do we need to relax these limitations a bit if that's the case?

I think this might already be in place for member info custom metadata, so we could have a separate ticket to adjust this globally. But, worth confirming now if it's needed or not.

Reply:
Yes would be good to have a standard limit for all fields where we expect PEM input. I noticed we may either have limits on the lower side (e.g. custom metadata) or have no limits set (e.g. MGM registration schema)

Reply:
In that case, can you raise a ticket under that epic to come back and bump all the defaults to account for this?

To get to a reasonable number, some calculations/research is needed to find out roughly what's a size of a PEM-formatted key for the different key types we support (e.g. RSA/ECDSA etc.) and add some extra headroom on top of that. 

P.S.: The limits for member side were put in place mostly for DoS protection AFAIK. No harm adding some limits for the group parameters custom metadata if none are present, although possibly not that critical given it's the network operator so effectively trusted by everyone.

Reply:
I've raised a ticket [here](https://r3-cev.atlassian.net/browse/CORE-15641)